lib/mayaUsd/render/vp2RenderDelegate/instancer.cpp

```diff
@@ -77,12 +82,18 @@ void HdVP2Instancer::_SyncPrimvars()
 
     // Use the double-checked locking pattern to check if this instancer's
     // primvars are dirty.
-    int dirtyBits = changeTracker.GetInstancerDirtyBits(id);
-    if (HdChangeTracker::IsAnyPrimvarDirty(dirtyBits, id)) {
+    HdDirtyBits dirtyBits = changeTracker.GetInstancerDirtyBits(id);
+    if (HdChangeTracker::IsAnyPrimvarDirty(dirtyBits, id)
+        || HdChangeTracker::IsInstancerDirty(dirtyBits, id)) {
```

Comment:
Does the change need conditional compilation on `HD_API_VERSION`?

Reply:
This should be fine without API guards too. `IsInstancerDirty()` and the `DirtyInstancer` dirty bit it uses are both present back to at least USD 19.11.